doc/tutorial.txt

```diff
@@ -272,6 +272,48 @@ inherit the affinity from the task level):
 	}
 }
 
+*** NUMA memory binding
+
+* nodes: Array of Integer. Define the NUMA binding of the thread. 
+Default value is all NUMA nodes of the system. An example :  "nodes" : [0, 2, 3]
+
+"nodes" can be specified at task level or phase level. As an example, below
+creates two threads. One thread will run with memory binding to nodes 2 and 3. The
+second task will run first phase with memory binding to nodes 0 and 1, 
+second phase with memory binding to node 2, 
+and the third phase with memory binding to nodes 4, 5, and 6 (it will inherit memory binding from the task level):
+
```

Comment:
I would prefer that you don't change the memory binding for the 3rd phase to be aligned with "policy", "priority", cgroups and utilization clamping instead of cpuset. I prefer to follow an event based policy than an inheritance which means that rt-app changes memory binding when there is a "nodes" event and don't do anything otherwise. "cpuset" is prone to mistake as it can change the affinity even if you haven't put anything in a phase.

This behavior will also simplify quite a lot the code as you don't need to create new string for default node affinity

Reply:
Sure , will do.